homeassistant/components/hassio/handler.py

```diff
@@ -249,6 +250,18 @@ async def async_update_core(
     )
 
 
+@bind_hass
+@_api_bool
+async def async_apply_suggestion(hass: HomeAssistant, suggestion_uuid: str) -> dict:
```

Comment:
How can it return `dict` when its decorated with `@_api_bool`?

Reply:
It can't. Was left from before. Fixed